src/SFA.DAS.AssessorService.Application.Api/Controllers/RegisterQueryController.cs

```diff
@@ -62,6 +62,27 @@ public async Task<IActionResult> GetAssessmentOrganisations()
             return Ok(await _mediator.Send(new GetAssessmentOrganisationsRequest()));
         }
 
+        [HttpGet("assessment-organisations/list", Name = "GetAssessmentOrganisationsList")]
+        [SwaggerResponse((int)HttpStatusCode.OK, Type = typeof(List<AssessmentOrganisationSummary>))]
+        [SwaggerResponse((int)HttpStatusCode.BadRequest, Type = typeof(IDictionary<string, string>))]
+        [SwaggerResponse((int)HttpStatusCode.InternalServerError, Type = typeof(ApiResponse))]
+        public async Task<IActionResult> GetAssessmentOrganisationsList()
+        {
+            _logger.LogInformation("Get Assessment Organisations List");
+            return Ok(await _mediator.Send(new GetAssessmentOrganisationsListRequest()));
+        }
+
+        [HttpGet("assessment-organisations/list/{ukprn}", Name = "GetAssessmentOrganisationsListByUkprn")]
+        [SwaggerResponse((int)HttpStatusCode.OK, Type = typeof(AssessmentOrganisationSummary))]
+        [SwaggerResponse((int)HttpStatusCode.BadRequest, Type = typeof(IDictionary<string, string>))]
+        [SwaggerResponse((int)HttpStatusCode.InternalServerError, Type = typeof(ApiResponse))]
+        public async Task<IActionResult> GetAssessmentOrganisationsListByUkprn(int ukprn)
+        {
+            _logger.LogInformation("Get Assessment Organisations List");
```

Comment:
Might be useful to include the `ukprn` in this log entry

Reply:
Fixed